includes/wc-template-functions.php

```diff
@@ -2694,3 +2694,106 @@ function wc_page_noindex() {
 function wc_get_theme_slug_for_templates() {
 	return apply_filters( 'woocommerce_theme_slug_for_templates', get_option( 'template' ) );
 }
+
+/**
+ * Gets and formats a list of cart item data + variations for display on the frontend.
+ *
+ * @since 3.3.0
+ * @param array $cart_item Cart item object.
+ * @param bool  $flat Should the data be returned flat or in a list.
+ * @return string
+ */
+function wc_get_cart_item_data( $cart_item, $flat = false ) {
```

Comment:
Should we rename this whilst at it? This function is getting a formatted list. `wc_get_formatted_cart_item_data`?